site/sidebar.md

```diff
@@ -4,6 +4,34 @@
     - [Homebrew](installation/homebrew/)
     - [Source](installation/source/)
     - [Binaries](installation/binaries/)
+- [Book](book/)
+    - [Getting Started](book/01-getting-started/00.md)
+        - [Quickstart](book/01-getting-started/01-quickstart.md)
+    - [Concepts](book/02-concepts/00.md)
+        - [Packages](book/02-concepts/01-packages.md)
+        - [Functions](book/02-concepts/02-functions.md)
+    - [Packages](book/03-packages/00.md)
+        - [Fetching An Existing Package](book/03-packages/01-fetching-an-existing-package.md)
+        - [Exploring A Fetched Package](book/03-packages/02-exploring-a-fetched-package.md)
+        - [Customizing A Package](book/03-packages/03-customizing-a-package.md)
+        - [Manual Editing](book/03-packages/04-manual-editing.md)
+        - [Automation Using Functions](book/03-packages/05-automation-using-functions.md)
+        - [Rebasing A Package](book/03-packages/06-rebasing-a-package.md)
+        - [Creating A New Package](book/03-packages/07-creating-a-new-package.md)
+        - [Publishing A Package To Git](book/03-packages/08-publishing-a-package-to-Git.md)
```

Comment:
Still not title case

Reply:
This is the output of golang's `strings.Title`
Replaced with a better title casing function in https://github.com/GoogleContainerTools/kpt/pull/1637